gateway/server.go

```diff
@@ -85,15 +85,15 @@ func main() {
 
 	r.HandleFunc("/system/alert", faasHandlers.Alert)
 
-	r.HandleFunc("/system/function/{name:[-a-zA-Z_0-9]+}", queryFunction).Methods("GET")
-	r.HandleFunc("/system/functions", listFunctions).Methods("GET")
-	r.HandleFunc("/system/functions", faasHandlers.DeployFunction).Methods("POST")
```

Comment:
I find the previous version much easier to read - but this seems like a no-harm change.